docs/Makefile

```
@@ -87,7 +89,7 @@ pa11y-install:
 install: $(VENVDIR)
 
 run: install
-	. $(VENV); $(VENVDIR)/bin/sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
+	. $(VENV); $(VENVDIR)/bin/sphinx-autobuild -b dirhtml --host $(SPHINX_HOST) --port $(SPHINX_HOST) "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
```

Thanks @SecondSkoll. I think you meant port here not host.
```suggestion
	. $(VENV); $(VENVDIR)/bin/sphinx-autobuild -b dirhtml --host $(SPHINX_HOST) --port $(SPHINX_PORT) "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
```

---

I did, thanks Charles! It's been a long day :)